pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -170,17 +188,14 @@ public void init(SegmentGeneratorConfig segmentCreationSpec, SegmentIndexCreatio
       Preconditions.checkState(dictEnabledColumn || !invertedIndexColumns.contains(columnName),
           "Cannot create inverted index for raw index column: %s", columnName);
 
-      IndexCreationContext.Common context = IndexCreationContext.builder()
-          .withIndexDir(_indexDir)
-          .withCardinality(columnIndexCreationInfo.getDistinctValueCount())
-          .withDictionary(dictEnabledColumn)
-          .withFieldSpec(fieldSpec)
-          .withTotalDocs(segmentIndexCreationInfo.getTotalDocs())
+      IndexCreationContext.Common context = IndexCreationContext.builder().withIndexDir(_indexDir)
```

Comment:
I’ve seen this change applied on several branches, because the original code is not formatted the way the automated formatting formats it. Which is more readable?

Reply:
true, maybe we can add some rules to not join lines that are already split?